### PR TITLE
Null safety Added

### PR DESCRIPTION
**Pull request by Intiserahmed**
**- What I did**

Upgraded at_hello world  (Complete and incomplete) to Null safety.

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: at_hello_world/server_demo_complete/pubspec.yaml [hunk 1]
```diff
@@ -4,17 +4,17 @@ publish_to: 'none'
 version: 0.1.0+1
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
   cupertino_icons: ^1.0.0
```
Comment (cpswan) [inline at_hello_world/server_demo_complete/pubspec.yaml]: Should cupertino_icons be bumped to 1.0.1 to get the minimal Null safety version?

Comment (Intiserahmed) [reply at_hello_world/server_demo_complete/pubspec.yaml]: Well it didn't show any error.  But I think I should include the latest null safety version of it. Thanks!

Comment (Intiserahmed) [reply at_hello_world/server_demo_complete/pubspec.yaml]: So it doesn't show any difference if I bumped to 1.0.3. I have checked.

Comment (cpswan) [reply at_hello_world/server_demo_complete/pubspec.yaml]: What difference would you expect to see?

Comment (Intiserahmed) [reply at_hello_world/server_demo_complete/pubspec.yaml]: Will Show  errors in the project?

File: at_cookbook/Complete_Version/.flutter-plugins-dependencies [hunk 2]
```diff
@@ -1 +1 @@
...
...andler","path_provider"]},{"name":"at_onboarding_flutter","dependencies":["file_picker","flutter_qr_reader","url_launcher","permission_handler","path_provider","webview_flutter","at_backupkey_flutter"]},{"name":"file_picker","dependencies":["flutter_plugin_android_lifecycle"]},{"name":"flutter_keychain","dependencies":[]},{"name":"flutter_plugin_android_lifecycle","dependencies":[]},{"name":"flutter_qr_reader","dependencies":[]},{"name":"path_provider","dependencies":["path_provider_macos","path_provider_linux","path_provider_windows"]},{"name":"path_provider_linux","dependencies":[]},{"name":"path_provider_macos","dependencies":[]},{"name":"path_provider_windows","dependencies":[]},{"name":"permission_handler","dependencies":[]},{"name":"share","dependencies":[]},{"name":"url_launcher","dependencies":["url_launcher_linux","url_launcher_macos","url_launcher_web","url_launcher_windows"]},{"name":"url_launcher_linux","dependencies":[]},{"name":"url_launcher_macos","dependencies":[]},{"name":"url_launcher_web","dependencies":[]},{"name":"url_launcher_windows","dependencies":[]},{"name":"webview_flutter","dependencies":[]}],"date_created":"2021-07-01 20:26:03.885666","version":"2.2.2"}
```
Comment (cpswan) [inline at_cookbook/Complete_Version/.flutter-plugins-dependencies]: This should be in .gitignore (and not here)

Comment (Intiserahmed) [reply at_cookbook/Complete_Version/.flutter-plugins-dependencies]: Noted

**Pull request closed without merging**